src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+will search for the specified workflow xml definition in this table.
+
+Below you will find the structure of both tables
+
+.Structure of the Submitted Job table
+[cols="1,1,1"]
+|===
+|Column name | Type | Max length
+
+| finished_time
+| Long
+| 40
+
+| in_error_time
+| Long
+| 40
+
+| job_id
+| Long
+| 40
+
+| name
+| String
+| 256
+
+| output
+| String
+| 4000
+
+| owner
+| String
+| 40
+
+| result
+| String
+| 40
+
+| session_id
+| String
+| 64
+
+| start_time
+| Long
+| 40
+
+| status
+| String
+| 16
+
+| submitted_time
+| Long
+| 40
+|===
+
+.Structure of the Workflow table
+[cols="1,1,1"]
+|===
+|Column name | Type | Max length
+
+| name
+| String
+| 256
+
+| file
+| File Attachment
+| 40
+
+|Cell in column 4, row 3
+|===
+
+[NOTE]
+All tables contain additional fields starting with sys_ . Those fields are automatically generated by ServiceNow when the table is created and can't be changed.
+
+Two Table Columns have been added to the Incident table:
+
+. ProActive Job Id
+. ProActive Session Id
+
+This allows to specify the sessionId of the user, and a job id at an Incident creation, binding it to a ProActive job.
+Other columns can be added in the Incident table to store logs, outputs or any needed information regarding a job.
+
+===== Flow Designer
+
+Flow Designer is a feature for automating processes in a single design environment.
+
+Actions and Core Actions are what composes a Flow and Subflow. The application comes with Actions ready to be used.
+These Actions use the Script Includes that we described earlier.
+
+An Action consists of:
+
+- Inputs: Variables accessible from all steps of the action
+- Action step(s): Process(es) using the provided inputs
+- Outputs: Variables that represent the results of the action. These results are available to other actions in a flow.
+
+ProActive's application Actions have been designed so that their inputs correspond to the related function called from
+the Scheduler Client Script Include to make it as linear as possible.
+
+.Available actions
+[.center, cols="1", width=25%]
+|===
+|Action name
+
+| Login to ProActive
+| Disconnect from ProActive
+| Submit job from catalog
+| Submit job from file
+| Wait for terminal status
+| Disconnect from ProActive
+|===
+
+Here is an example with the "Submit From Catalog" action. Principles are the same for other actions
+
+.Inputs of the Submit From Catalog action
+image::servicenow-action-inputs.png[align="center"]
+
+The Action inputs are accessible by all steps of the Action
+
+Inputs of this step are bound to the Action inputs. This is represented by a pill
+in the "Value" column.
+
+.Inputs of the Submit From Catalog script step
+image::servicenow-step-inputs.png[align="center"]
+
+The action is composed of a single Script step. A script step is JavaScript code to execute within a reusable action.
+While most core actions and steps fit common use cases, a Script step enables to execute behavior not satisfied by the core steps.
```

Comment:
a Script step enables to execute a behavior that is not satisfied by the core steps